tensorflow/python/keras/activations.py

```diff
@@ -369,6 +369,29 @@ def tanh(x):
   return nn.tanh(x)
 
 
+@keras_export('keras.activations.tanhexp')
+@dispatch.add_dispatch_support
+def tanhexp(x):
+  """TanhExponential activation function, `tanhexp(x) = x * tanh(exp(x))`.
```

Comment:
Rephrase to "TanhExp activation function" for consistency. Please fix throughout.

Reply:
updated activations.py with suggested changes